CHANGELOG.md

```diff
@@ -0,0 +1,171 @@
+# Changelog
+
+## [Unreleased]
+
+### Added
+* add `9` to travis (@ralphtheninja)
+
+### Changed
+* upgrade to `abstract-leveldown@4.0.0` (@ralphtheninja)
+
+### Removed
+* remove `DeferredLevelDOWN.prototype._isBuffer`, no longer needed since we use `Buffer.isBuffer()` (@ralphtheninja)
+
+### Fixed
+* defer `approximateSize()` separately (@ralphtheninja)
+* fixed broken link in `README` and clean up old `.jshintrc` (@ralphtheninja)
+
+**Historical Note** `abstract-leveldown@4.0.0` dropped `approximateSize()` so we needed to handle deferring this method only for stores that has support for it, instead of handing it for all possible stores.
```

Comment:
- "stores that has support for it" => "stores that support it"
- "instead of handing it for all possible stores" is somewhat redundant, maybe leave it out.